dev-python/gentoo-common/gentoo-common-0.ebuild

```
@@ -0,0 +1,30 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Common files shared by Python implementations in Gentoo"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:Python"
+S=${WORKDIR}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86"
+
+src_install() {
+	insinto /usr/lib/python
+	# https://peps.python.org/pep-0668/
+	newins - EXTERNALLY-MANAGED <<-EOF
+		[externally-managed]
+		Error=
+		 System-wide Python installation in Gentoo should be maintained
```

```suggestion
		 The system-wide Python installation in Gentoo should be maintained
```
Since there is only one system-wide installation?